R/s3.get.R

```diff
@@ -78,3 +81,20 @@ s3.get <- function (path, bucket.location = "US", verbose = FALSE, debug = FALSE
   }
   ans
 }
+
+## Given an s3cmd path and a bucket location, will construct a flag
+## argument for s3cmd.  If it looks like the s3cmd is actually
+## pointing to an s4cmd, return empty string as s4cmd doesn't
+## support bucket location.
+bucket_location_to_flag <- function(s3cmd_binary_path, bucket_location) {
+  if (grepl("s4cmd", s3cmd_binary_path)) {
+    if (bucket_location != "US") {
+        warning(paste0("Ignoring non-default bucket location ('",
+                       bucket_location,
+                       "') in s3mpi::s3.get since s4cmd was detected",
+                       "-- this might be a little slower but is safe to ignore."));
+    }
+    return("")
```

Comment:
R will always return the last evaluated thing by default, so explicit `return` isn't necessary except for early returns.